File: go-client-sdk/pkg/models/shared/invalidparam.go
package shared

// InvalidParam
// Information about an input parameter pass to the API call that could not be parsed.
type InvalidParam struct {
	DataType      string `json:"dataType"`
	InputCellType string `json:"inputCellType"`
	ParamName     string `json:"paramName"`
	ParamValue    string `json:"paramValue"`
}

File: typescript-client-sdk/src/sdk/models/shared/invalidparam.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


// InvalidParam
/** 
 * Information about an input parameter pass to the API call that could not be parsed.
**/
export class InvalidParam extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "dataType" })
  dataType: string;

  @SpeakeasyMetadata()
  @Expose({ name: "inputCellType" })
  inputCellType: string;

  @SpeakeasyMetadata()
  @Expose({ name: "paramName" })
  paramName: string;

  @SpeakeasyMetadata()
  @Expose({ name: "paramValue" })
  paramValue: string;
}